### DISCUSSION TITLE
The LeftFluent Mode text is incomplete

### DISCUSSION THREAD

**Discussion by ghost** [lepoco/wpfui#968] [Q&A]
The text is not displayed completely, which property should I set?
![1](https://github.com/lepoco/wpfui/assets/26182676/8fc2e3fc-af87-4cfd-b2c4-2dd7072f6fbf)

**Comment by Mithicor** [comment]
I haven't used the "LeftFluent" Mode, but in the "Left" mode I use the 'OpenPaneLength' property to adjust for text getting cut off.


**EDIT:** Looking further into the Styles, it looks like the width and height of the 'FluentLeft' style is hardcoded. The style can be found here:

https://github.com/lepoco/wpfui/blob/main/src/Wpf.Ui/Controls/NavigationView/NavigationViewConstants.xaml
